db/schema.rb

```diff
@@ -16,7 +16,7 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "oauth_access_grants", force: true do |t|
+  create_table "oauth_access_grants", force: :cascade do |t|
```

Comment:
Any idea why these changed? Probably doesn't matter but just curious as we're not making any database-related changes here.